### PR TITLE
Fix for #50

### LINKED ISSUE #50: Support Plural* in noun table
For instance see the word [Gelb](https://de.wiktionary.org/wiki/Gelb):
```
{{Deutsch Substantiv Übersicht
|Genus=n
|Nominativ Singular=Gelb
|Nominativ Plural=Gelbs
|Nominativ Plural*=Gelbtöne
|Genitiv Singular=Gelbs
|Genitiv Plural=Gelbs
|Genitiv Plural*=Gelbtöne
|Dativ Singular=Gelb
|Dativ Plural=Gelbs
|Dativ Plural*=Gelbtönen
|Akkusativ Singular=Gelb
|Akkusativ Plural=Gelbs
|Akkusativ Plural*=Gelbtöne
|Bild 1=Zeichen_306_-_Vorfahrtstraße,_StVO_1970.svg|200px|1|ein Verkehrszeichen mit Gelbanteilen
|Bild 2=Andwil Oberarnegg Briefkasten.jpg|200px|1|Gelb ist die Farbe vieler Postbetriebe, hier ein Schweizer Briefkasten
}}
```

This uses `Plural*` for a second plural option.

### PR DESCRIPTION
**Pull request by highsource**
This fixes #50 by adding label.endsWith("*") check.
Unit test is also included using the word Gelb.

### CODE REVIEW

**Comment by highsource** [comment]
CLA not signed yet.

**Comment by highsource** [comment]
I've asked for a confirmation for CCLA, but this may take my employer very long time (huge corp).
Should I maybe sign the ICLA first?

**Comment by chmeyer** [comment]
Thanks for all your changes. ICLA would be fine for merging. CCLA is recommended, but it depends on what status the work has for your company and how they deal with OSS projects in general.

File: src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/DEWordFormHandler.java [hunk 1]
```diff
@@ -183,9 +188,12 @@ public boolean processBody(final String textLine, final ParsingContext context)
 	protected void extractNounTable(final WiktionaryWordForm wordForm, String label) {
 //		if ("die Koseworte".equals(wordForm.getWordForm()))
 //			System.out.println("X");
-		if (label.endsWith("1") || label.endsWith("2")
-				|| label.endsWith("3") || label.endsWith("4"))
-			label = label.substring(0, label.length() - 1).trim();
+		for (String labelSuffix: NOUN_TABLE_LABEL_SUFFIXES) {
```
Comment (chmeyer) [inline src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/DEWordFormHandler.java]: maybe a RegEx, e.g., `\s\d*\*?\*?$` would be more efficient?

**Comment by highsource** [comment]
I'm working to get CCLA but it may take pretty long time.

File: src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/DEWordFormHandler.java (hunk 1 above)
Comment (highsource) [reply src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/DEWordFormHandler.java]: I'd leave it like this for now. Most probably I'll rework this for #57 anyway.

Personally, I'm not a fan of regular expressions. I would not be able to understand what `\s\d*\*?\*?$` does without checking the documentation.

**Comment by highsource** [comment]
I've signed and e-mailed the ICLA.

**Pull request merged**